PlayCover/Model/PlayApp.swift

```diff
@@ -107,6 +107,19 @@ class PlayApp: BaseApp {
         }
     }
 
+    var aliasDirectory: URL {
+        return FileManager.default.homeDirectoryForCurrentUser
+            .appendingPathComponent("Applications")
+            .appendingPathComponent("PlayCover")
+    }
+
+    var aliasURL: URL {
+        return FileManager.default.homeDirectoryForCurrentUser
```

Comment:
Since the first part of the url is the same, it might as well use `aliasDirectory` and be a lazy variable.